Source/Core/VideoCommon/RenderBase.h

```diff
@@ -211,7 +211,7 @@ class Renderer
   virtual u32 AccessEFB(EFBAccessType type, u32 x, u32 y, u32 poke_data);
   virtual void PokeEFB(EFBAccessType type, const EfbPokeData* points, size_t num_points);
 
-  virtual u16 BBoxRead(int index) = 0;
+  u16 BBoxRead(int index);
   virtual void BBoxWrite(int index, u16 value) = 0;
```

Comment:
BBoxWrite and BBoxFlush should also follow this change as well to make the API consistent.

It's not consistent for BBoxWrite to be renamed to BBoxWriteImpl's in all the interfaces but BBoxRead and BBoxFlush are as is.

Reply:
Well, it's redundant for those, but I guess if you want for the sake of consistency.

Reply:
Thanks!

Reply:
Done. I also split the changes up for organizational purposes.